src/components/modeler/Modeler.vue

```diff
@@ -337,30 +337,20 @@ export default {
     async pasteElements() {
       if (this.copiedElements) {
         await this.addClonedNodes(this.copiedElements);
+        await this.$nextTick();
+        await this.paperManager.awaitScheduledUpdates();
         this.$refs.selector.selectElements(this.findViewElementsFromNodes(this.copiedElements));
```

Comment:
can you comment out 340 and 341 and add an `await` to 342 since it is an async function to see if the problem gets solved that way? Either way, you have to add the `await` to line 342

Reply:
The `await this.$nextTick();` can be removed.
The `await this.paperManager.awaitScheduledUpdates();` is required from ticket https://processmaker.atlassian.net/browse/FOUR-7998
adding await to `this.$refs.selector.selectElements`